subprojects/core-api/src/main/java/org/gradle/work/DisableCachingByDefault.java

```diff
@@ -41,7 +40,6 @@
  *
  * @since 7.0
  */
-@Incubating
```

Comment:
We should add something to the release notes as well, right?

Reply:
Added.